app/api.py

```diff
@@ -791,3 +798,21 @@ def get_element_text(self, doc, tag_name):
             return text.encode('utf-8')
         else:
             return None
+
+    #Send hit to GA
+    def gamp(e,ec):
+        ga_cid = uuid.uuid4()
+        params = urllib.urlencode({
```

Comment:
Can you add a comment to point to the document of Measurement Protocol so that people can understand what these parameters are?